### ISSUE TITLE
Can not install solc on the Mac.

### ISSUE THREAD

**Issue by Skyge** [crytic/solc-select#25]
I am trying to install solc following our document, 
```bash
docker pull trailofbits/solc-select
docker run --read-only -i --rm --entrypoint='/bin/sh' trailofbits/solc-select:latest -c 'cat /usr/bin/install.sh' | bash -e
```
when I run first command, it can pull some images, but for the second commands, nothing happens, so what should I do?

**Comment by Skyge** [comment]
Maybe something wrong with my system, I have used a soft link to solve this problem.

**Comment by Skyge** [comment]
I tried the approach **Installation from Source on OS X**, and I got the same result: nothing happened, so what is wrong?
Besides, is there a necessary to download all `solc` versions? As a matter of fact, there is a recommending for compiling contracts, if we use `^0.4.x`, it should use the latest version in the `0.4`(now is `0.4.26`), and it is also recommend to use a specific version, such as `0.4.24` rather than `^0.4.24`, so once I have chosen a version, I will use it all the time, so we can use a cache to save the `solc` file, if we do not have this version, we can download it.If I want to remove it, just remove the cache file, but now, I download all versions, and I want to remove them, I really do not know how to do it.

**Comment by disconnect3d** [comment]
TLDR: The problem is that we can't write to `/usr/bin/solc` on OS X [due to SIP](https://apple.stackexchange.com/questions/237778/cant-write-to-usr-even-with-sudo).

And we do it with:
```
docker run --read-only -i --rm --entrypoint='/bin/sh' trailofbits/solc-select:latest -c 'cat /usr/bin/install.sh' | bash -e
```

as this cats the install script and pipes it to bash on the host machine.

This script begins with the content from this script: https://github.com/crytic/solc-select/blob/master/scripts/install.sh#L51 and exactly this line fails as we have no permission to write to `/usr/bin`.

A workaround is to do this steps:
```
docker run --read-only -i --rm --entrypoint='/bin/sh' trailofbits/solc-select:latest -c 'cat /usr/bin/install.sh' > a.sh
sed s+\'/usr/bin/solc\'+\'/usr/local/bin/solc\'+ a.sh > b.sh
chmod a+x b.sh
./b.sh
chmod a+x /usr/local/bin/solc
rm a.sh b.sh
```

**Comment by Skyge** [comment]
@disconnect3d  Yeah, it works now, thanks! so is there a necessary to change the default installation directory to avoid such errors just like mine?

**Comment by disconnect3d** [comment]
> so is there a necessary to change the default installation directory to avoid such errors just like mine?

Yeah, we need to make such change in the repo.
